BridgeAppSDK.xcodeproj/project.pbxproj

```diff
@@ -2193,7 +2201,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
```

Comment:
iOS 10 is coming out this week. Set BridgeAppSDK to minimum deployment of 8.4 and the sample app to min target of 9.3 as per Apple's recommendations for min targets.